tools/readme_config_includer/generator.go

```diff
@@ -54,7 +54,8 @@ func extractIncludeBlock(txt []byte, includesEx *regexp.Regexp, root string) *in
 		}
 		include := filepath.FromSlash(string(inc[1]))
 		// Make absolute paths relative to the include-root if any
-		if filepath.IsAbs(include) {
+		// Need to also check for leading '\' on Windows systems
+		if filepath.IsAbs(include) || strings.HasPrefix(include, "\\") {
```

Comment:
How about checking `string(inc[1])`, i.e. the path before conversion as we know to expect unix-style paths there.